### PR TITLE
fix: height on payment method view

### PR DESCRIPTION
**Pull request by dangelovska**

### CODE REVIEW

File: brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelOne/KYCBasicViewController.swift [hunk 1]
```diff
@@ -21,6 +21,7 @@ class KYCBasicViewController: BaseTableViewController<KYCCoordinator,
         return L10n.Account.personalInformation
     }
     private var isValid = false
+    private var isPickCountryPressed = false
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelOne/KYCBasicViewController.swift:24]: What does this solve exactly?

Comment (dangelovska) [reply brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelOne/KYCBasicViewController.swift:24]: preventing multiple opening of the country selector view

Comment (kenanmamedoff) [reply brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelOne/KYCBasicViewController.swift:24]: was that a reported bug? this just isn't a good way of solving things

Comment (dangelovska) [reply brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelOne/KYCBasicViewController.swift]: no, it wasn't reported

**Review by kenanmamedoff — APPROVED** [review]

**Pull request merged**